pkg/smclient/client.go

```diff
@@ -179,6 +180,32 @@ func (client *serviceManagerClient) ListPlatforms() (*types.Platforms, error) {
 	return platforms, err
 }
 
+// ListOfferings returns service offerings of brokers registered in Service Manager
+func (client *serviceManagerClient) ListOfferings() (*types.ServiceOfferings, error) {
+	serviceOfferings := &types.ServiceOfferings{}
+	err := client.list(serviceOfferings, "/v1/service_offerings")
```

Comment:
We can use the constants from service manager for the resources URLs. Applies for all other places where URLs are used. They should be in package `web`